ci/images/ci-image-jdk16/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:focal-20210119
+
+ADD setup.sh /setup.sh
+ADD get-jdk-url.sh /get-jdk-url.sh
+ADD get-docker-url.sh /get-docker-url.sh
+RUN ./setup.sh java8 java16
```

Comment:
To elaborate on this. I think that executing things with the actual JDK is always better than working with the toolchain support. Hence specifying the same version everywhere else except for this JDK 16 image. As soon as everything works on JDK 16 (e.g. the Gradle Plugin) I would also suggest to specify `RUN ./setup.sh java16 java16`. 

This is based on my personal experience while working on the JDK upgrades in the past. The real deal always beats the toolchain/artificial setup.